### DISCUSSION TITLE
Google不用怕地位被威脅了! 紐約學校禁用ChatGpt

### DISCUSSION THREAD

**Discussion by HuangHsinTzu** [1101-GS4719-C/Python-Community-GS4719-C#1088] [General]
ChatGPT聊天機器人最近在全球爆紅，雖然掀起風潮，負面爭議也伴隨而來，美國邱約是教育局開始禁止公立學校的設備和網路使用ChatGPT

禁止的原因在於防止學生作弊，以及認為學生過度依賴ChatGPT會妨礙思考，還有ChatGPT的答案不一定準確
不過妞約禁令限制公立學校設備和網路，學生個人設備也不受限

ChatGPT崛起的確對Google造成不小壓力，甚至傳出Google內部為此拉起「紅色警戒」，上緊發條加速發展AI聊天機器人產業

我是覺得Google不必過度擔心，像現在雖然台灣的教育部沒有頒布相關的禁令，學生遇到問題或想蒐集資料時還是大都仰賴Google
因為Google的資料比較豐富，查詢資料本來就要多方參考，只依據ChatGPT單方面給的答案感覺還是不太可靠
https://3c.ltn.com.tw/news/51923

**Comment by chia-kai-chang** [comment]
我不認同美國紐約公立學校的做法

隨著時代演進，工具本來也會隨之進步，現在應該要教大家如何正確使用現在工具，而不是直接禁止

不然，時間拉回到工業命時期，怎麼不禁止汽車? 反正人力走路一樣可以到達目的地。而且這則禁令只有禁止學校的電腦不可使用ChatGPT但是學生手機，家裡電腦一樣可以使用。禁令沒有甚麼實質意義，看來比較像是政治公關

>《官場現形記·第四六回》：「臣是天朝的大臣，應該按照國家的制度辦事。什麼火車、輪船，走的雖快，總不外乎奇技淫巧。」

**Comment by wenchi9023** [comment]
站在紐約教育局的立場，我想他們知道就算ChatGPT被禁止，學生還是可以上Google找參考資料。因此我認為教育局這麼做是為了制止ChatGPT聊天機器人越做越精準，以免學生未來只依靠ChatGPT上提供的答案進行作答；若禁止ChatGPT的話至少還讓學生還能從Google上的廣泛資料中保有獨立思考、做最後批判的能力。
